Tests/AspNetCore2.Tests/AspNetCore2.Tests.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
This is in order to be able to test the fix. Since the `IsEssential` property doesn't exist in ASP.NET Core 2.0, the `MakeEssential` method would have no effect on that framework.